### DISCUSSION TITLE
Cannot able to open MOOSE website

### DISCUSSION THREAD

**Discussion by avtarsinghh1991** [idaholab/moose#22883] [Q&A Modules: General] [answered]
Hello MOOSE Experts,

I am trying to open the MOOSE website on the NREL network and could able to do so.
I talked with NREL IT support and they replied as follows:

"I think you're going to have to find a support contact for MOOSE. We (NREL) are not blocking access to it, but it looks like their site is not responding to connections from NREL at all. It appears to be accessible from elsewhere on the internet, so access from NREL should work too. In case they ask, NREL users would appear to be coming from an IP address on this network: ************"

I am not sure if I can share the IP address here on public platform.

Any suggestions.

Best,
Avtar

**Comment by loganharbour** [comment, marked as answer]
@asingh-mit, go ahead and e-mail me at logan.harbour@inl.gov in regards to this

**Reply by avtarsinghh1991** [reply]
Sent you an email. Thank you very much.
